.github/workflows/update-flu-catalogue.yml

```diff
@@ -1,4 +1,4 @@
-name: 'update nextstrain.org/influenza with latest flu datasets in s3'
+name: 'update nextstrain.org dataset listing resources on s3'
 
 on:
   schedule:
```

Comment:
I think as part of this PR we can reduce the frequency of updates to 1 hour here. We should also add a line to the `/staging` page indicating that it may take an hour for updates to show (similar to the message at `/groups`).